experimenter/experimenter/nimbus_ui_new/templates/common/footer.html

```diff
@@ -1,5 +1,6 @@
-<footer class="w-full mt-2 flex justify-center items-center">
-    <div class="flex flex-col justify-center text-spaceCadet items-center w-5/6 p-4 border-t border-azure">
-      © Mozilla Corporation 2023
-    </div>
+<footer class="text-center my-4">
+  <hr />
```

Comment:
You don't need the `/`. Thats a JSX / XHTML thing.

Reply:
Ooo let's see if we can enforce that with the djlinter.